src/Lib/Models/CanvasItem.vala

```diff
@@ -25,15 +25,17 @@ public enum Akira.Lib.Models.CanvasItemType {
     TEXT
 }
 
-public interface Akira.Lib.Models.CanvasItem : Goo.CanvasItem {
+public interface Akira.Lib.Models.CanvasItem : Goo.CanvasItemSimple, Goo.CanvasItem {
     public static int global_id = 0;
 
     public abstract string id { get; public set; }
```

Comment:
Do you need to specify a `public` setter? I've never seen this before.

Reply:
No I don't have actually, I was following the compiler's suggestions. In the end even without public or protected the code worked the same, so I removed it!